access/types/forms.py

```diff
@@ -685,10 +698,6 @@ def grade_checkbox(self, configuration, value, hints=None, name=''):
             else:
                 correct = (non_neutral_count / 2.0 - wrong_answers) / (non_neutral_count / 2.0)
 
-        # Add note of multiple correct answers.
```

Comment:
Did this logic go somewhere else, or do we no longer add this one?

Reply:
From what I saw, all the checkbox-questions say that "choose the ones that apply etc.", or "what are the things that so so and so", so the question seems always to imply that there are multiple choices to be selected. Also it didn't seem to translate at all, so it looked a bit funny with the Finnish courses among Finnish hints, so I thought it was only confusing. 

![image](https://user-images.githubusercontent.com/43036333/87749831-69476d80-c802-11ea-8d1a-d817c0b1869b.png)

But hmm.. Now that I give it a second thought, that might be a good default-hint to still have. I will add that back in my current implementation as well.

Reply:
And this has been added back :)